README.md

```diff
@@ -31,6 +31,32 @@ Create the API Key in the [DataRobot Developer Tools](https://app.datarobot.com/
 By default, all components use `datarobot_default` connection ID.
 
 
+## Creating DataRobot preconfigured Connections
+
+You can create preconfigured Connections to store and manage credentials 
+that can be used together with Airflow Operators to replicate connection 
+on DataRobot side ([Data connections](https://docs.datarobot.com/en/docs/data/connect-data/stored-creds.html)).
+Currently, supported the next types of credentials:
+
+* `DataRobot Basic Credentials` - to store login/password pairs
+* `DataRobot GCP Credentials` - to store Google Cloud Service account key
+* `DataRobot AWS Credentials` - to store AWS access keys
+* `DataRobot Azure Storage Credentials` - to store Azure Storage secret
+* `DataRobot OAuth Credentials` - to store OAuth tokens
+* `DataRobot JDBC DataSource` - to store JDBC connection attributes
+
+After creating preconfigured Connections using Airflow UI or Airflow API [Managing Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
+it can be used with GetOrCreateCredentialOperator or GetOrCreateDataStoreOperator
+to replicate it on DataRobot side and retrieve corresponding credentials_id
+or datastore_id. 
```

Comment:
```suggestion
to replicate it on DataRobot side and retrieve corresponding `credentials_id`
or `datastore_id`. 
```

Can you also make a warning that credentials are only created one time -- if the value of the credential is changed in Airflow secret storage, they must be manually updated in DataRobot Credentials Manager to keep them in sync.

Reply:
I will add this in separate a commit